apps/web/src/utils/url.ts

```diff
@@ -1,5 +1,5 @@
 export function openInNewTab(url: string) {
-  return window.open(url, '_blank', 'noreferrer noopener');
+  window.open('https://google.com', '_blank', 'noreferrer noopener');
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please remove -- assuming this was a test‏